lib/Cache.js

```diff
@@ -125,25 +128,27 @@ class Cache {
 				if (typeof value === 'undefined') { continue; }
 
 				for (const prop in slice.filter) {
+
 					if (
 						!this._checkFilter(entity, prop, slice.filter[prop])
 					) {
 						continue keyloop; //eslint-disable-line
 					}
 				}
-				if (this.knowledge[key].contains(value)) {
-					keyValues[key] = { value, item: entity };
-				}
 
-				if (slice.link) { for (const linkType in slice.link) {
-					const dKey = this.getDualKey(slice, linkType);
+				keyValues[key] = { value, item: entity };
```

Comment:
Correct, where are you creating this new point range ([6, 6])?

Without doing that, I would prefer to skip adding this item altogether, I can't see any harm in doing that.

Reviewed, everything else is good.